downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -63,6 +63,60 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 11 (2201.11.0)
 #### CLI
 
 #### OpenAPI tool
+- Introduced the `flatten` sub-command, which flattens the OpenAPI contract file by moving 
+  all the inline schemas to the components section. The output is a modified OpenAPI contract.
+
+  ```bash
```

Comment:
```suggestion
  ```
```
Remove bash keyword. Otherwise the syntax highlighter will render it as bash script, which results in incorrect syntax highlighting.